lib/src/models/user.dart

```diff
@@ -2,15 +2,17 @@ import 'profile.dart';
 
 class User {
   final int id;
+  final String? email;
   final String firstName;
   final String lastName;
+  final String? phoneNumber;
   final List<Profile> profiles;
-  final bool isGuest;
```

Comment:
Removed unused `isGuest` and added `email` and `phoneNumber` because they are used by the SSO when we upgrade a plan.